db/app/app.js

```diff
@@ -0,0 +1,16 @@
+const express = require("express");
+const { getTopics } = require("./controller");
+const { handlePsqlErrors, handleCustomErrors, handleServerErrors } = require("./errors");
+
+const app = express();
+
+app.use(express.json());
+
+app.get("/api/topics", getTopics)
+
+
+app.use(handlePsqlErrors);
```

Comment:
Also this is boilerplate code that is not currently being used. Only add code when you are TDDing it's use and not before.